diff -aurN gnome-doc-utils-0.4.3.orig/gnome-doc-utils.make gnome-doc-utils-0.4.3/gnome-doc-utils.make
--- gnome-doc-utils-0.4.3.orig/gnome-doc-utils.make	2005-10-12 06:29:37.000000000 +0200
+++ gnome-doc-utils-0.4.3/gnome-doc-utils.make	2005-10-13 15:06:47.000000000 +0200
@@ -692,10 +692,6 @@
 	  echo "$(INSTALL_DATA) $$omf $(DESTDIR)$(OMF_DIR)/$(DOC_MODULE)/$$omf"; \
 	  $(INSTALL_DATA) $$omf $(DESTDIR)$(OMF_DIR)/$(DOC_MODULE)/$$omf; \
 	done
-	@if test "x$(_ENABLE_SK)" = "xtrue"; then \
-	  echo "scrollkeeper-update -p $(DESTDIR)$(_sklocalstatedir) -o $(DESTDIR)$(OMF_DIR)/$(DOC_MODULE)"; \
-	  scrollkeeper-update -p "$(DESTDIR)$(_sklocalstatedir)" -o "$(DESTDIR)$(OMF_DIR)/$(DOC_MODULE)"; \
-	fi;
 
 install-doc-dsk:
 	echo install-dsk
